recipe/migrations/arrow_cpp500.yaml

```
@@ -0,0 +1,7 @@
+__migrator:
+  build_number: 1
+  kind: version
+  migration_number: 1
+arrow_cpp:
+- 5.0.0
```

Alternatively to the `keyadd` migrator 

```suggestion
- 3.0.0
- 4.0.1
- 5.0.0
```

---

The order is important here, I run into this with the nodejs migration already.
```suggestion
- 5.0.0
- 4.0.1
- 3.0.0
```